src/redis-cli.c

```diff
@@ -81,6 +81,7 @@ static redisContext *context;
 static struct config {
     char *hostip;
     int hostport;
+    struct timeval connect_timeout;
```

Comment:
Minor thing but why put the new field here? Does not look to fit, so makes sense to add at the end unless there is already a group of related fields.

Reply:
Not minor at all - I actually remember thinking about this and it made sense (for me) to put it there because it is related to TCP just like the two above it. That said, I'll move it the end if that makes more sense to you.